package.json

```diff
@@ -70,13 +70,13 @@
     "ml-array-xy-equally-spaced": "^1.2.1",
     "ml-baseline-correction-regression": "^1.0.0",
     "ml-conrec": "^3.2.1",
-    "ml-gsd": "^12.0.0",
+    "ml-gsd": "12.1.0",
     "ml-spectra-processing": "^11.12.0",
     "ml-stat": "^1.3.3",
     "multiplet-analysis": "^2.0.0",
     "nmr-correlation": "^2.3.3",
     "nmr-parser": "^1.9.1",
-    "nmr-processing": "^9.0.3",
+    "nmr-processing": "^9.0.3-pre.1661576806",
```

Comment:
Just making sure this isn't merged with the preversion.

Reply:
@lpatiny did you test the behavior of nmrium with this pre-release of nmr-processing? can I merge the PR in nmr-processing?

Reply:
I didn't see any new issue with this version.